e2e_test/batch/keyword/user.slt

```diff
@@ -0,0 +1,16 @@
+statement ok
+SET RW_IMPLICIT_FLUSH TO true;
+
+statement ok
+create table t (id int, user varchar);
+
+statement ok
+insert into t values (1, 'abc');
+
+query I
+select id, "user", 'user' from t;
```

Comment:
What about including unquoted `user` as well, to better illustrate ths differences?

Reply:
`user` is not deterministic, so I prefer to exclude it

Reply:
Done